install.sh

```diff
@@ -330,9 +330,12 @@ nvm_do_install() {
   BASH_OR_ZSH=false
 
   if [ -z "${NVM_PROFILE-}" ] ; then
-    echo "=> Profile not found. Tried ${NVM_PROFILE} (as defined in \$PROFILE), ~/.bashrc, ~/.bash_profile, ~/.zshrc, and ~/.profile."
+    if [ -n "${PROFILE}" ]; then
+      local TRIED_PROFILE
```

Comment:
doesn't this `local` have to appear outside the `if`, if it's referenced outside it?

Reply:
I think it's still in the same function?

Reply:
For clarity, even if it's not required, it'd be nice to declare the var at a high enough scope to eclipse all uses.

Reply:
So would moving it to above current line 333 be okay?

Reply:
yup, seems good!